solr/solrj/src/java/org/apache/solr/common/util/StrUtils.java

```diff
@@ -388,4 +388,19 @@ public static String stringFromReader(Reader inReader) throws IOException {
       return stringWriter.toString();
     }
   }
+
```

Comment:
Really?   Sigh...   we don't have this already!???

Reply:
We've long used Apache common-lang3's `StringUtils`, which offers this and all kinds of really nice String-utilities.

But recently we decided to put any use of `StringUtils` (or maybe commons-lang altogether?) in our forbidden-api checking, and so we've been adding to the Solr `StrUtils` class slowly since then.  I'm sure there's a good reason for the forbidden-api, but not knowing the "why" I'm always irked when I hit this and have to write yet-another function that we could be getting "for free" from any number of libraries.

And then precommit fails because we're doing a reference-equality check, so we have to put in a Suppression for that. Just so so so much fun...

![old-man-cloud](https://github.com/apache/solr/assets/9030708/eb7eb480-ef40-43c9-b517-b977becf38f7)